cli.js

```diff
@@ -25,10 +25,28 @@ if (args.includes('-v')) {
   args[args.indexOf('-v')] = '--version';
 }
 
+if (args.includes('-l')) {
+  args[args.indexOf('-l')] = '--loader';
+}
+
 if (args.includes('-n')) {
   args[args.indexOf('-n')] = '--no-module';
 }
 
+let loaderFilename = false;
+let createLoaderFile = false;
+
+if (args.includes('--loader')) {
```

Comment:
Seems that this if clause is missing `}` at the end. Otherwise it swallowed the whole code that comes after (and this missing `}` breaks `cli.js` as a whole).

Reply:
Sorry about that. There was a merge problem and it looks like I didn't resolve it properly. Should be fixed now.